components/organisms/AnnouncementHistoryTable.jsx

```
@@ -0,0 +1,668 @@
+        }
+      />
+
+      {/* Filters */}
+      <Card>
+        <CardHeader>
+          <CardTitle className="text-lg">Filters</CardTitle>
+        </CardHeader>
+        <CardContent>
+          <div className="grid gap-4 md:grid-cols-2">
+            <div className="space-y-2">
+              <label className={cn(poppins_500.className, "text-sm")}>
```

ESLint

[error] A form label must be associated with a control.

(jsx-a11y/label-has-associated-control)